controls/cis_rhel7.yml

```diff
@@ -567,3 +567,255 @@ controls:
     - l1_server
     - l1_workstation
     automated: no
+
+  - id: 2.1.1
+    title: Ensure xinetd is not installed (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: yes
+    rules:
+    - package_xinetd_removed
+
+  - id: 2.2.1.1
+    title: Ensure time synchronization is in use (Manual)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: no
+
+  - id: 2.2.1.2
+    title: Ensure chrony is configured (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: yes
+    rules:
+    - chronyd_specify_remote_server
+    - var_multiple_time_servers=rhel
+    - chronyd_run_as_chrony_user
+
+  - id: 2.2.1.3
+    title: Ensure ntp is configured (Automated)
+    levels:
+    - l1_server
+    - l1_workstation
+    automated: yes
+    rules:
+    - service_ntpd_enabled
+    - ntpd_configure_restrictions
+    - ntpd_specify_remote_server
```

Comment:
In previous version we opinionated the profile toward `chrony`, as we had a rule to install it.
With the item `2.2.1.1` being manual, the choice whether to use ntpd or chrony is put on the sysadmin.

The `chrony` rules have `platform: chrony`.
But the only `ntpd' rule with `platform: ntp` is `ntpd_configure_restrictions`, could you add platform ntp to the other ntp rules so that they are only run when it is installed?

Reply:
fixed